framework/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Yii Framework 2 Change Log
 - Enh #11139: `yii\validators\EachValidator` injects specific attribute value in error message parameters (silverfire)
 - Enh #11187: migrate command now generates phpdoc for table migrations (Faryshta)
 - Enh: Added `StringHelper::countWords()` that given a string returns number of words in it (samdark)
+- Enh: migrate command can create foreign keys.
```

Comment:
how about `fk-{{%source_table}}-source_column` its very unlikely the same column in the same table have two different foreignt keys.

but then again, my mainq uestion is if that sintax is ok for the SQL parser in the yii2 db schema or if it will throw an error when using table prefixes.